### ISSUE TITLE
Returning list of paths in Edmond Karp's

### ISSUE THREAD

**Issue by Azzaare** [JuliaGraphs/LightGraphsFlows.jl#24]
I need to extract the set of paths composing the max-flow returned by Edmond Karp's algorithm (or any augmenting paths method). Do you mind if I add this set of paths to the `edmonds_karp_impl` function as a returned value?

If it is OK, I will make a push request soon.

**Comment by sbromberger** [comment]
Can this be derived from the `flow_matrix`? If so, it might be better as a standalone function.

**Comment by Azzaare** [comment]
You're correct, I was probably being lazy. Then I will write down some flow_decomposition function for any flow matrix and make a PR.

**Comment by matbesancon** [comment]
let us know if you need help on the PR on the way :)
